spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/server/db/migration/OracleSmokeTest.java

```diff
@@ -15,22 +15,50 @@
  */
 package org.springframework.cloud.dataflow.server.db.migration;
 
+import java.util.Locale;
+
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.testcontainers.containers.OracleContainer;
+import org.testcontainers.utility.DockerImageName;
+import uk.org.webcompere.systemstubs.environment.EnvironmentVariables;
+import uk.org.webcompere.systemstubs.jupiter.SystemStub;
+import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
 
-import org.springframework.test.context.DynamicPropertyRegistry;
-import org.springframework.test.context.DynamicPropertySource;
-
+import org.springframework.core.log.LogAccessor;
 
 /**
  * Basic database schema and JPA tests for Oracle XE.
  *
  * @author Corneil du Plessis
+ * @author Chris Bono
  */
+@ExtendWith(SystemStubsExtension.class)
 public class OracleSmokeTest extends AbstractSmokeTest {
+
+	private static final LogAccessor LOG = new LogAccessor(OracleSmokeTest.class);
```

Comment:
In follow on commit I am going to abstract this out so that it can be used easily on any "oracle" tests.